sway-core/src/language/ty/declaration/declaration.rs

```diff
@@ -915,7 +912,8 @@ impl TyDecl {
                 decl_engine.get_configurable(decl_id).visibility
             }
             TyDecl::ConstGenericDecl(_) => {
-                todo!("Will be implemented by https://github.com/FuelLabs/sway/issues/6860")
+                // const generics do not have visibility
```

Comment:
Similarly here, in case we opt for always the same single line explanation within the `unreachable!()`.